pkgpanda/docker.windows/dcos-builder/Dockerfile

```diff
@@ -59,5 +66,6 @@ RUN C:\TEMP\vs_buildtools.exe --quiet --wait --norestart --nocache --installPath
     || IF "%ERRORLEVEL%"=="3010" EXIT 0
 
 # Start developer command prompt with any other commands specified.
-
-ENTRYPOINT C:\BuildTools\Common7\Tools\VsDevCmd.bat && powershell.exe -file
\ No newline at end of file
+# note: vsdevcmd.bat builds for x86 by default, use these options for amd64:

```

Comment:
I don't know anything about those two scripts, unfortunately. I think the `&&` is necessary because this is intended to run the script defined in the entrypoint before running the command provided at run-time. (For reference, the image created by this Dockerfile is used in `pkgpanda/build/__init__.py`, in `_build()`.)